datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -459,6 +498,32 @@ impl ParquetExec {
         ParquetExecBuilder::new(file_scan_config)
     }
 
+    /// Convert this `ParquetExec` into a builder for modification
+    pub fn into_builder(self) -> ParquetExecBuilder {
+        // list out fileds so it is clear what is being dropped
```

Comment:
```suggestion
        // list out fields so it is clear what is being dropped
```

Reply:
I wonder if the dropped fields will be rebuilt during the `built()` function?

Reply:
Yes they are -- I will make that clear in comments